src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/IndividualDto.java

```diff
@@ -0,0 +1,370 @@
+
+    public void setUraAddressCareOf(String uraAddressCareOf) {
+        this.uraAddressCareOf = uraAddressCareOf;
+    }
+
+    public String getUraAddressCountry() {
+        return uraAddressCountry;
+    }
+
+    public void setUraAddressCountry(String uraAddressCountry) {
+        this.uraAddressCountry = uraAddressCountry;
+    }
+
+    public String getUraAddressLocality() {
+        return uraAddressLocality;
+    }
+
+    public void setUraAddressLocality(String uraAddressLocality) {
+        this.uraAddressLocality = uraAddressLocality;
+    }
+
+    public String getUraAddressPoBox() {
+        return uraAddressPoBox;
+    }
+
+    public void setUraAddressPoBox(String uraAddressPoBox) {
+        this.uraAddressPoBox = uraAddressPoBox;
+    }
+
+    public String getUraAddressPostalCode() {
+        return uraAddressPostalCode;
+    }
+
+    public void setUraAddressPostalCode(String uraAddressPostalCode) {
+        this.uraAddressPostalCode = uraAddressPostalCode;
+    }
+
+    public String getUraAddressPremises() {
+        return uraAddressPremises;
+    }
+
+    public void setUraAddressPremises(String uraAddressPremises) {
+        this.uraAddressPremises = uraAddressPremises;
+    }
+
+    public String getUraAddressRegion() {
+        return uraAddressRegion;
+    }
+
+    public void setUraAddressRegion(String uraAddressRegion) {
+        this.uraAddressRegion = uraAddressRegion;
+    }
+
+    public AddressDto getUsualResidentialAddress() {
+        AddressDto address = new AddressDto();
+        address.setPropertyNameNumber(uraAddressPremises);
+        address.setLine1(uraAddressLine1);
+        address.setLine2(uraAddressLine2);
+        address.setCounty(uraAddressRegion);
+        address.setLocality(uraAddressLocality);
+        address.setCountry(uraAddressCountry);
+        address.setCareOf(uraAddressCareOf);
+        address.setPoBox(uraAddressPoBox);
+        address.setPostcode(uraAddressPostalCode);
+        return address;
+    }
+
+    public String getDateBecameInterestedPersonDay() {
+        return dateBecameInterestedPersonDay;
+    }
+
+    public void setDateBecameInterestedPersonDay(String dateBecameInterestedPersonDay) {
+        this.dateBecameInterestedPersonDay = dateBecameInterestedPersonDay;
+    }
+
+    public String getDateBecameInterestedPersonMonth() {
+        return dateBecameInterestedPersonMonth;
+    }
+
+    public void setDateBecameInterestedPersonMonth(String dateBecameInterestedPersonMonth) {
+        this.dateBecameInterestedPersonMonth = dateBecameInterestedPersonMonth;
+    }
+
+    public String getDateBecameInterestedPersonYear() {
+        return dateBecameInterestedPersonYear;
+    }
+
+    public void setDateBecameInterestedPersonYear(String dateBecameInterestedPersonYear) {
+        this.dateBecameInterestedPersonYear = dateBecameInterestedPersonYear;
+    }
+
+    public String getDateBecameInterestedPerson() {
+        if (type.equals("Individual Interested Person")
```

Comment:
similar comment to the one in CorporateDto, instead of having the hardcoded string here, should reference it from the BeneficialOwnerType enum